CONTRIBUTING.md

```diff
@@ -40,14 +40,14 @@ mkdocs serve
 ## Submitting a PR
 1. [Fork the repo](https://github.com/dyvenia/viadot/fork)
 2. [Install](./README.md#installation) and [configure](./README.md#configuration) `viadot`  
-    __Note__: In order to run tests, you will also need to install dev dependencies in the `viadot_2` container with `pip install -r requirements-dev.txt --user`
+    >__Note__: In order to run tests, you will also need to install dev dependencies in the `viadot_2` container with `pip install -r requirements-dev.txt --user`
```

Comment:
`!!! note` looks better in MkDocs, see https://github.com/dyvenia/nesso/blob/merge_cli/docs/cli/getting_started/getting_started.md

Reply:
But `CONTRIBUTING.md` is not in mkdocs

Reply:
ah you're right